test/integration/Aave.t.sol

```diff
@@ -55,161 +101,236 @@ contract AaveIntegrationTest is BaseTest
         _pairs.push(_stablePair);
     }
 
+    function setUp() public
```

Comment:
```suggestion
    function setUp() external
```

Favour external when public is not necessary (i.e. func is not called within contract).

Reply:
okay so we should ensure this even for tests as well. Fixed and noted.